src/Hint/Naming.hs

```diff
@@ -110,3 +146,9 @@ replaceNames :: Data a => [(String,String)] -> a -> a
 replaceNames rep = transformBi f
     where f (Ident _ x) = Ident an $ fromMaybe x $ lookup x rep
           f x = x
+replaceNamesNew :: Data a => [(String,String)] -> a -> a
+replaceNamesNew rep = transformBi replace
+    where 
+        replace :: Hs.RdrName -> Hs.RdrName
+        replace (Hs.Unqual (Hs.unsafePrettyPrint -> name)) = Hs.Unqual $ Hs.mkOccName Hs.srcDataName $ fromMaybe name $ lookup name rep
```

Comment:
The part I'm unsure about!

0. Is it fine to only match on `Unqual` here?
1. Is it fine to construct `OccName`s with `srcDataName` here?